### PR TITLE
feat: Add Atom Feed for flags

### PR DESCRIPTION
**Pull request by btrd**

### CODE REVIEW

File: apps/cf_atom_feed/lib/flags.ex [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  end
+
+  defp fetch_flags() do
+    DB.Repo.all(
+      from(
+        flag in Flag,
+        order_by: [desc: flag.inserted_at],
+        left_join: action in assoc(flag, :action),
+        left_join: comment in assoc(action, :comment),
+        left_join: statement in assoc(comment, :statement),
```
Comment (Betree) [inline apps/cf_atom_feed/lib/flags.ex]: You don't need to load the statement, `action.video_id` already contains the info you need

File: apps/db/test/support/factory.ex [hunk 2]
```diff
@@ -118,6 +118,7 @@ defmodule DB.Factory do
       target_user: build(:user),
       type: :create,
       entity: :comment,
+      comment: build(:comment),
```
Comment (Betree) [inline apps/db/test/support/factory.ex]: This is what make the test fail. Is it really required?

Comment (btrd) [reply apps/db/test/support/factory.ex]: For my test yes, without I doesn't have the proper data in my db. But it's maybe not required here.

**Comment by Betree** [comment]
Looks good to me, thanks @btrd !

**Comment by Betree** [comment]
@btrd Your PR has been merged, tested on staging and will be included in the next release. We're going to plug this feed on our Discord so the community will be notified when a comment is flagged on the website!

Do you have a Twitter handle we could use to thank you publicly :slightly_smiling_face:  ?

**Comment by btrd** [comment]
Cool cool cool ! https://twitter.com/btrd_me

**Pull request merged**